sdk/Dockerfile

```diff
@@ -26,7 +26,7 @@ RUN apk update && \
     apk add linux-headers && \ 
     apk add perl && \
     apk add bash && \
-    pip3 install conan
+    pip3 install -r requirements.txt
```

Comment:
Won't work, file is not available in container context, yet

Reply:
Reverted back to "simple" fix - we need to check anyway if this container is still needed.
And lifecycle mgm needs also being introduced